src/render.js

```diff
@@ -36,39 +36,48 @@ export function update(index = 0, startTime = 0) {
   }
 }
 
-export default function render(get) {
+function addToQueue(event) {
+  const target = event.composedPath()[0];
+  if (target === event.currentTarget) {
+    if (!queue[0]) {
+      requestAnimationFrame((() => update()));
+    }
+    if (queue.indexOf(target) === -1) {
+      queue.push(target);
+    }
+  }
+}
+
+export default function render(get, options = { shadowRoot: true }) {
```

Comment:
There may be a bug here in the future. We want `{shadowRoot: true}` to be the default when no shadowRoot property is provided, but someone using the following code would _not_ get a shadowRoot:
```javascript
define('my-elem', {
  render: render(() => html`
    <p>I expected shadowDOM but this is in light DOM.</p>
  `, { someOtherOption: true })
};
```
Should the code be something like this:
```javascript
const DEFAULT_RENDER_OPTS = { shadowRoot: true };
export default function render(get, options = {}) {
  const renderOpts = Object.assign({}, DEFAULT_RENDER_OPTS, options);
  ...
}
```

Reply:
That's right. I changed it, but with simpler syntax using spread operator.